gef-ui/server/controllers/unissued-facilities/index.js

```diff
@@ -209,16 +208,19 @@ const postChangeUnissuedFacility = async (req, res) => {
     req.success = {
       message: `${body.facilityName} is updated`,
```

Comment:
Looks like we don't need to modify global `req` object. Can we set text `${body.facilityName} is updated` directly to req.flash call.

Reply:
Changed to req.flash('success', {
  message: `${body.facilityName} is updated`,
});